recipe/test_script_exist.bat

```diff
@@ -0,0 +1,25 @@
+@echo on
+
+REM As a workaround for GDAL apps unexpectedly returning error codes, this
+REM script checks only for the existence of the .py files below.
+
+echo Test existence of gdalattachpct.py...
+if not exist %PREFIX%\\Library\\bin\\gdalattachpct.py exit 1
```

Comment:
We could tidy up with `%LIBRARY_BIN%` but let's see if this works first.

Reply:
That does not seem to make any difference

Reply:
As I said, it's just tidier.  If the file isn't one place, it isn't the other because they point the same place.  If the test is failing, the file presumably really isn't installed there.